<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppMainClass</class>
 <widget class="QMainWindow" name="AppMainClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1270</width>
    <height>862</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI Emoji</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>4th View</string>
  </property>
  <property name="windowIcon">
   <iconset resource="AppMain.qrc">
    <normaloff>:/AppMain/resources/SEB 4 empty.png</normaloff>:/AppMain/resources/SEB 4 empty.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">
QDockWidget{
	border:1px solid gray;
}</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::TabShape::Rounded</enum>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::DockOption::AllowNestedDocks|QMainWindow::DockOption::AllowTabbedDocks|QMainWindow::DockOption::AnimatedDocks|QMainWindow::DockOption::GroupedDragging</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="replay_control">
        <property name="styleSheet">
         <string notr="true">QPushButton#replay_control{
	background:none;
	border:none;
}

QPushButton#replay_control:hover{
	width:20px;
	height: 20px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="QIcon::ThemeIcon::MediaPlaybackStart"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="seeker_bar">
        <property name="styleSheet">
         <string notr="true">QSlider#seeker_bar::handle:horizontal{
	width: 8px;
    height: 8px;
     margin: -4px 0; /* Centers the handle vertically */
     border-radius: 8px;
     background: #3498db;
}

/*
QSlider#seeker_bar::handle:horizontal:hover{
 	  width: 24px;
     height: 24px;
     margin: -12px 0;
     border-radius: 12px;
     background: #2980b9;
}
*/

QSlider#seeker_bar::add-page:horizontal {
    background: grey;
}

QSlider#seeker_bar::sub-page:horizontal {
    background: red;
}

</string>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
        <property name="sliderPosition">
         <number>500</number>
        </property>
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="live_mode_button">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#live_mode_button{
	background-color: rgb(202, 0, 0);
}</string>
        </property>
        <property name="text">
         <string>Live</string>
        </property>
        <property name="icon">
         <iconset resource="AppMain.qrc">
          <normaloff>:/AppMain/resources/icons8-live-50.png</normaloff>:/AppMain/resources/icons8-live-50.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1270</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionConnect"/>
    <addaction name="actionStart_Capture"/>
    <addaction name="actionStop_Capture"/>
    <addaction name="actionPing_Modulees"/>
   </widget>
   <widget class="QMenu" name="menuTracker">
    <property name="title">
     <string>Tracker</string>
    </property>
    <addaction name="actionLaunch_4"/>
   </widget>
   <widget class="QMenu" name="menuPlotter">
    <property name="title">
     <string>Plotter</string>
    </property>
    <addaction name="actionLaunch_3"/>
   </widget>
   <widget class="QMenu" name="menuStats">
    <property name="title">
     <string>Stats</string>
    </property>
    <addaction name="actionLaunch_2"/>
   </widget>
   <widget class="QMenu" name="menuATracker">
    <property name="title">
     <string>ATracker</string>
    </property>
    <addaction name="actionLaunch"/>
    <addaction name="actionOn_Air"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionSave_Perspective"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuUpdates">
    <property name="title">
     <string>Updates</string>
    </property>
    <addaction name="actionCheck_for_Updates"/>
    <addaction name="actionVersion"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTracker"/>
   <addaction name="menuPlotter"/>
   <addaction name="menuStats"/>
   <addaction name="menuATracker"/>
   <addaction name="menuWindow"/>
   <addaction name="menuUpdates"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBarMain">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTracker"/>
   <addaction name="actionPlotter"/>
   <addaction name="actionATracker"/>
   <addaction name="actionSport_Events"/>
   <addaction name="action3D_Model"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="styleSheet">
    <string notr="true">
QDockWidget{
	border:1px solid gray;
}</string>
   </property>
   <property name="windowTitle">
    <string>Object Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="objectsList">
    <property name="minimumSize">
     <size>
      <width>299</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="object_list">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>279</width>
          <height>555</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_4">
   <property name="windowTitle">
    <string>logs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_4">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>90</height>
     </size>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionClear_All"/>
   <addaction name="actionAdd_Distance"/>
   <addaction name="actionSwitch_Camera_View"/>
   <addaction name="actionStart_Live_Data_Capture"/>
   <addaction name="actionPause_Output_Stream"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5">
   <property name="windowTitle">
    <string>Player Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <property name="minimumSize">
     <size>
      <width>299</width>
      <height>0</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Properties</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Track ID</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Position</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>X:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_2">
              <property name="styleSheet">
               <string notr="true">#lineEdit_2{
	border-left-color: rgb(255, 0, 0);
	border-left-width: 5px;
}</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Y:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_3">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_7">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Fielder</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_3">
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Coordinates</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="3">
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Annotate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Highlight</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Annotation Text:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit">
              <property name="readOnly">
               <bool>false</bool>
              </property>
              <property name="placeholderText">
               <string>Enter text that will appear on the graphic</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_2">
              <property name="text">
               <string>Activate</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Position</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBox">
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Fielder</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Bowler</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Batter Facing</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Batter</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Umpire</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Update</string>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
          </property>
          <property name="shortcut">
           <string>Ctrl+S</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFeature::DockWidgetClosable|QDockWidget::DockWidgetFeature::DockWidgetFloatable|QDockWidget::DockWidgetFeature::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Available IDs</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>100</height>
     </size>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QListView" name="listView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionTracker">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaRecord"/>
   </property>
   <property name="text">
    <string>Tracker</string>
   </property>
   <property name="toolTip">
    <string>live tracker</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionPlotter">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::InputMouse"/>
   </property>
   <property name="text">
    <string>Plotter</string>
   </property>
   <property name="toolTip">
    <string>player position plotter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionATracker">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaRecord"/>
   </property>
   <property name="text">
    <string>ATracker</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionSport_Events">
   <property name="icon">
    <iconset resource="AppMain.qrc">
     <normaloff>:/AppMain/resources/database-icon</normaloff>:/AppMain/resources/database-icon</iconset>
   </property>
   <property name="text">
    <string>Sport Events</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action3D_Model">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::CameraVideo"/>
   </property>
   <property name="text">
    <string>3D Model</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditClear"/>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAdd_Distance">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::ListAdd"/>
   </property>
   <property name="text">
    <string>Add Distance</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSwitch_Camera_View">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::CameraVideo"/>
   </property>
   <property name="text">
    <string>Switch Camera View</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionStart_Live_Data_Capture">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaRecord"/>
   </property>
   <property name="text">
    <string>Start Live Data Capture</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCheck_for_Updates">
   <property name="text">
    <string>Check for Updates</string>
   </property>
  </action>
  <action name="actionVersion">
   <property name="text">
    <string>Version</string>
   </property>
  </action>
  <action name="actionSave_Perspective">
   <property name="text">
    <string>Save Perspective</string>
   </property>
  </action>
  <action name="actionLaunch">
   <property name="text">
    <string>Launch</string>
   </property>
  </action>
  <action name="actionOn_Air">
   <property name="text">
    <string>On Air</string>
   </property>
  </action>
  <action name="actionLaunch_2">
   <property name="text">
    <string>Launch</string>
   </property>
  </action>
  <action name="actionLaunch_3">
   <property name="text">
    <string>Launch</string>
   </property>
  </action>
  <action name="actionLaunch_4">
   <property name="text">
    <string>Launch</string>
   </property>
  </action>
  <action name="actionStart_Capture">
   <property name="text">
    <string>Start Capture</string>
   </property>
  </action>
  <action name="actionStop_Capture">
   <property name="text">
    <string>Stop Capture</string>
   </property>
  </action>
  <action name="actionPing_Modulees">
   <property name="text">
    <string>Ping Modules</string>
   </property>
  </action>
  <action name="actionPause_Output_Stream">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::MediaPlaybackPause"/>
   </property>
   <property name="text">
    <string>Pause Output Stream</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="AppMain.qrc"/>
 </resources>
 <connections/>
</ui>
